Source/Scene/Cesium3DTileset.js

```diff
@@ -523,6 +523,14 @@ define([
         this.loadSiblings = defaultValue(options.loadSiblings, false);
 
         /**
+         * For point cloud tiles, whether to scale points as they get closer to the camera.
+         *
+         * @type {Boolean}
+         * @default true
+         */
+        this.pointAttenuation = true;
```

Comment:
Mention this in CHANGES.md.

Are you sure this is the only field you want to expose, e.g., no parameters to change the size?

Are you also sure that it belongs here and not on the style?  This seems out of place and more tightly coupled.

Reply:
> Are you also sure that it belongs here and not on the style? This seems out of place and more tightly coupled.

I don't think it makes sense to be on the style, all 3D Tiles-conformant engines would need to support point attenuation in their engines. I think of this as similar to the color blend mode for tilesets, just an engine option to tweak the visuals.

Reply:
> Are you sure this is the only field you want to expose, e.g., no parameters to change the size?

@AnimatedRNG also suggested this. I might make them private to start since I expect the attenuation to be more involved in the future.

Reply:
> @AnimatedRNG also suggested this. I might make them private to start since I expect the attenuation to be more involved in the future

Could be fine to be public and then deprecate everything when we have the new method; otherwise, I think users will ask for these properties right away.